packages/graphql-server/src/ee/resolvers/phDeployment.ts

```diff
@@ -533,6 +533,55 @@ export const destroy = async (root, args, context: Context) => {
   return {id};
 };
 
+export const destroyByGroup = async (
+  context: Context,
+  group: { id: string; name: string },
+  dryrun: boolean
+) => {
+  const { crdClient } = context;
+  const phDeployments = await crdClient.phDeployments.list();
+  const transformedPhDeployments = phDeployments.map(item => ({
+    id: item.metadata.name,
+    groupId: item.spec.groupId,
+    groupName: item.spec.groupName || '',
```

Comment:
Is `groupName` will receive a null or undefined value?

Reply:
No, i think. The groupName is copied from the original `transform` function. But I agree to remove the `|| ''` .

Reply:
I didn't mention *removing* any value, just want to confirm has value or not.

Reply:
The value is always available if creating from graphql api. The only case that the value is absent is we create the k8s resource from kubectl.

Reply:
the || '' has been removed. Because even the `groupName` is undefined, it should still work.